.vscodeignore

```diff
@@ -15,3 +15,6 @@ prettier.config.js
 contributing.md
 ContributorAgreement.txt
 .travis.yml
+.eslintignore
+.eslintrc.js
+.c8rc
```

Comment:
does `coverage/` need to be added here too?

Reply:
@scnwwu agreed, will add that to the list here.